testsuite/layers-expansion/pom.xml

```diff
@@ -705,6 +705,7 @@
                                                 <layer>microprofile-openapi</layer>
                                                 <layer>microprofile-reactive-streams-operators</layer>
                                                 <layer>microprofile-reactive-messaging</layer>
+                                                <layer>microprofile-reactive-messaging-amqp</layer>
```

Comment:
Please file a JIRA to add a provisioning execution with only the layer. All layers need an individual execution.

Reply:
Fixed in https://github.com/wildfly/wildfly/pull/17491